app/src/main/java/com/fueledbycaffeine/mivote/injection/ApiModule.kt

```diff
@@ -0,0 +1,52 @@
+package com.fueledbycaffeine.mivote.injection
+
+import com.squareup.moshi.Moshi
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.android.components.ActivityComponent
+import io.michiganelections.api.Api
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.converter.moshi.MoshiConverterFactory
+import timber.log.Timber
+
+@Module
+@InstallIn(ActivityComponent::class)
+class ApiModule {
```

Comment:
Shouldnt the stuff in here be part of the api gradle module?

Reply:
I somewhat agree and I attempted that. I ran into a few things...
1. `@InstallIn` only works in an android application module. So I tried moving everything _but_ the `provideApiService` into the api module.. that led me to
2. Because the api is a java library I dont believe that Timber will work. If you have a preference for a different way of logging I would be willing to give that a shot.

Reply:
I tried making a `RetrofitModule` dagger module in the API gradle module that had all the moshi/retrofit stuff, and include it inside the `ApiModule` dagger module but it complained that it also needed `@InstallIn` so that was a no go. 

kindof annoying tbh.

RE: 2, you would have the app which has a place to log things to supply the logger to the API module.

but i dont care enough about it to continue trying to sort it out for an MVP.

Reply:
What would you like me to do? :/